testdata/LoadAccountsTest/testupdateAccountsFromFile/accounts_load.tsv

```diff
@@ -0,0 +1,2 @@
+

```

Comment:
I don't understand this change.  It looks to me like it's adding a new file which just has two blank lines.  Is that the intent?

Reply:
It was a test file for a unit test that I must have removed.
It was to test for empty accounts load file.